qiskit/wrapper/configrc/_settings.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Module that holds the runtime location of qiskitrc file.
+"""
+
+QISKIT_RC_FILE = None
```

Comment:
Actually, this is not a setting but shared state for the functions in [`_configrc.py`](https://github.com/QISKit/qiskit-core/pull/547/files#diff-7964b0eebe1f71cbbea9d0d929667e33R1). I'm not a friend of using OOP when it is not necessary, so a module works form me. This variable should be in `_configrc.py` as a module-level.

Reply:
I think this reply is misplaced but it is fair.